src/VisualizationTests/HelixImageComparisonTests.cs

```diff
@@ -0,0 +1,332 @@
+            var node2 = ViewModel.CurrentSpace.Nodes.Where(x => x.Name.Contains("colorpoints")).FirstOrDefault();
+            node1.IsFrozen = true;
+            node2.IsFrozen = true;
+            RenderCurrentViewAndCompare(MethodBase.GetCurrentMethod().Name);
+        }
+
+        [Test]
+        public void lines()
+        {
+            OpenVisualizationTest(@"imageComparison\linecolors.dyn");
+            RunCurrentModel();
+            RenderCurrentViewAndCompare(MethodBase.GetCurrentMethod().Name);
+        }
+
+        [Test]
+        public void linesIsolated()
+        {
+            OpenVisualizationTest(@"imageComparison\linecolors.dyn");
+            RunCurrentModel();
+            View.BackgroundPreview.ViewModel.IsolationMode = true;
+            RenderCurrentViewAndCompare(MethodBase.GetCurrentMethod().Name);
+        }
+
+        [Test]
+        public void linesSelected()
+        {
+            OpenVisualizationTest(@"imageComparison\linecolors.dyn");
+            RunCurrentModel();
+            var node1 = ViewModel.CurrentSpace.Nodes.Where(x => x.Name.Contains("coloredLines")).FirstOrDefault();
+            var node2 = ViewModel.CurrentSpace.Nodes.Where(x => x.Name.Contains("regularLines")).FirstOrDefault();
+            DynamoSelection.Instance.Selection.Add(node1);
+            DynamoSelection.Instance.Selection.Add(node2);
+            RenderCurrentViewAndCompare(MethodBase.GetCurrentMethod().Name);
+        }
+
+        [Test]
+        public void linesFrozen()
+        {
+            OpenVisualizationTest(@"imageComparison\linecolors.dyn");
+            RunCurrentModel();
+            var node1 = ViewModel.CurrentSpace.Nodes.Where(x => x.Name.Contains("coloredLines")).FirstOrDefault();
+            var node2 = ViewModel.CurrentSpace.Nodes.Where(x => x.Name.Contains("regularLines")).FirstOrDefault();
+            node1.IsFrozen = true;
+            node2.IsFrozen = true;
+            RenderCurrentViewAndCompare(MethodBase.GetCurrentMethod().Name);
+        }
+
+        #endregion
+
+        #region surfaces
+        [Test]
+        public void WavySurfaceRender()
+        {
+            OpenVisualizationTest(@"imageComparison\wavysurface.dyn");
+            RunCurrentModel();
+            RenderCurrentViewAndCompare(MethodBase.GetCurrentMethod().Name);
+        }
+        #endregion
+
+        #region SpecialRenderPackages
+        [Test]
+        public void directManipulator()
+        {
+            var pointOriginNode =
+            new DSFunction(Model.LibraryServices.GetFunctionDescriptor("Point.ByCoordinates@double,double,double"));
+
+            var command = new Dynamo.Models.DynamoModel.CreateNodeCommand(pointOriginNode, 0, 0, true, false);
+            Model.ExecuteCommand(command);
+            RunCurrentModel();
+            DynamoSelection.Instance.Selection.Add(pointOriginNode);
+            RenderCurrentViewAndCompare(MethodBase.GetCurrentMethod().Name);
+        }
+
+        #endregion
+
+    }
+}
```

Comment:
NIT: Let's use `PascalCase` for naming all test methods, as that's the C# convention.

Reply:
Updated